### PR TITLE
MM-15312 - Updating tooltip UX guidelines

### PR DESCRIPTION
**Pull request by asaadmahmood**
@jasonblais Have a look.
I also notice that we are not doing things properly in some places, like "Add Reaction" should be changed to "Add reaction" etc, but that can be looked into in as a separate ticket.

Ticket: https://mattermost.atlassian.net/browse/MM-15312

### CODE REVIEW

File: source/developer/fx-guidelines.rst [hunk 1]
```diff
@@ -248,6 +248,16 @@ The color of the on_hover state should be the same as in active state, with opac
 |
 ---------------------------
 
+Display: Tooltips
```
Comment (jasonblais) [inline source/developer/fx-guidelines.rst]: Propose moving below `Display: Instructions` to keep the design guidelines section in alphabetical order.

File: source/developer/fx-guidelines.rst [hunk 2]
```diff
@@ -248,6 +248,16 @@ The color of the on_hover state should be the same as in active state, with opac
...
+---------------------------
+
+Tooltips text can be title case or sentence case.
```
Comment (jasonblais) [inline source/developer/fx-guidelines.rst]:
```suggestion
Tooltip text may be title case or sentence case depending on their context.
```

File: source/developer/fx-guidelines.rst [hunk 3]
```diff
@@ -248,6 +248,16 @@ The color of the on_hover state should be the same as in active state, with opac
...
+
+- Tooltips will be title case if the tooltip is a noun, e.g. Pinned Posts, or Flagged Posts.
```
Comment (jasonblais) [inline source/developer/fx-guidelines.rst]:
```suggestion
- Tooltips should be title case if the tooltip is a noun, e.g. "Pinned Posts", or "Flagged Posts".
```

File: source/developer/fx-guidelines.rst [hunk 4]
```diff
@@ -248,6 +248,16 @@ The color of the on_hover state should be the same as in active state, with opac
...
+- Tooltips will be sentence case if they are announcing an action, e.g. Flag for follow up, or Create new private channel.
```
Comment (jasonblais) [inline source/developer/fx-guidelines.rst]:
```suggestion
- Tooltips should be sentence case if they are declaring an action, e.g. "Flag for follow up", or "Create new private channel".
```

File: source/developer/fx-guidelines.rst (hunk 2 above)
Comment (asaadmahmood) [reply source/developer/fx-guidelines.rst]: Shouldn't this be, depending on its context, as a "Tooltip" is singular.
Or we can use "Tooltips" and then use "their".

File: source/developer/fx-guidelines.rst (hunk 1 above)
Comment (asaadmahmood) [reply source/developer/fx-guidelines.rst]: Done.

File: source/developer/fx-guidelines.rst (hunk 2 above)
Comment (asaadmahmood) [reply source/developer/fx-guidelines.rst]: @jasonblais ^

Comment (jasonblais) [reply source/developer/fx-guidelines.rst]: @asaadmahmood Yes, good catch. 

```
Tooltip text may be title case or sentence case depending on its context.
```

Comment (asaadmahmood) [reply source/developer/fx-guidelines.rst]: Makes sense.

**Comment by asaadmahmood** [comment]
@jasonblais Done.

**Pull request merged**